cszczepaniak/cribbly-backend
# Add team to division endpoint

# What I added (link any issues addressed)
Closes #26 

- I also got on a roll and added the ```GetById``` endpoint

# How to test it

Run ``` PUT https://localhost:5001/api/division/<division ID>/add```

Body (team ID):

```
{
    "Id": int
}
```
- See that a full, updated Division object is returned
- Optionally, also run ```GET https://localhost:5001/api/division/<division ID>``` and see that you get the same result as the previous call
